packages/core/src/hooks/useNavigateFromLoginScreen.ts

```diff
@@ -0,0 +1,29 @@
+import { useDemoMode, useLanguage } from "../store";
+import { useAppVersion, useMadConfig, useNavigateToMainRoute } from "../store/mad-config";
+import { useReleaseNotesVersion } from "../store/release-notes";
+import { getNavigationRouteForLoginScreen } from "../utils/getNavigationRouteForLoginScreen";
+import { useCoreStackNavigation } from "./useCoreStackNavigation";
+
+export const useNavigateFromLoginScreen = () => {
+    const appVersion = useAppVersion();
+    const { lastDisplayedReleaseNotesVersion } = useReleaseNotesVersion();
+    const { isEnabled: isDemoModeEnabled } = useDemoMode();
+    const navigateToMainRoute = useNavigateToMainRoute();
+    const navigation = useCoreStackNavigation();
+    const {
+        language: { supportedLanguages, skipOnboarding },
```

Comment:
Should we get `supportedLanguages` or both from `useLanguage` instead?

Reply:
supportedLanguages is returned from `useLanguage` just in case it's of any importance to the people who need the `useLanguage` hook. The source will always be `useMadConfig`. Might as well get it from the source.

`skipOnboarding` does not make sense to include in `useLanguage`